src/_scss/components/tooltips/_customTooltipContent.scss

```diff
@@ -76,6 +76,9 @@
         font-style: italic;
     }
 }
+.award-summary__new-category{
```

Comment:
This block can be removed from this file because it's present in searchSidebar.scss and that is applying to the elements on the page